cli/testdata/scorecard/policy-library/policies/templates/k8s_dump_review_template.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: k8sdumpreview
+spec:
+  crd:
+    spec:
+      names:
+        kind: K8sDumpReview
+  targets:
+    - target: admission.k8s.gatekeeper.sh
```

Comment:
For the future, it would be good to add a test which actually looks at data so we can be sure it passes through properly.